pkgs/os-specific/linux/it87/default.nix

```
@@ -7,15 +7,15 @@
 
 stdenv.mkDerivation rec {
   name = "it87-${version}-${kernel.version}";
-  version = "unstable-2024-06-09";
+  version = "unstable-2025-03-27";
```

```suggestion
  version = "0-unstable-2025-03-27";
```
It must start with a digit